packages/flutter/lib/src/rendering/paragraph.dart

```diff
@@ -735,6 +735,11 @@ class RenderParagraph extends RenderBox
     }
   }
 
+  /// An estimate of the height of a line in the text. See [TextPainter.preferredLineHeight].
+  /// This does not required the layout to be updated.
```

Comment:
required => require

Reply:
Or maybe this? "Accessing this value does not cause the layout to be updated."

Reply:
Removed this method per @GaryQian 's recommendation.